ios/FluentUI/Avatar/AvatarTokens.swift

```diff
@@ -179,15 +179,15 @@ open class AvatarTokens: ControlTokens {
     open var backgroundDefaultColor: DynamicColor {
         switch style {
         case .default, .group:
-            return .init(light: globalTokens.neutralColors[.white], dark: globalTokens.brandColors[.primary].dark)
+            return aliasTokens.colors[.background1]
         case .accent:
-            return globalTokens.brandColors[.primary]
+            return aliasTokens.colors[.brandBackground1]
         case .outlined:
-            return .init(light: globalTokens.neutralColors[.grey94], dark: globalTokens.neutralColors[.grey26])
+            return aliasTokens.colors[.background5]
         case .outlinedPrimary:
-            return .init(light: globalTokens.brandColors[.tint40].light, dark: globalTokens.neutralColors[.grey26])
+            return aliasTokens.colors[.brandBackground4]
         case .overflow:
-            return aliasTokens.backgroundColors[.neutral4]
```

Comment:
nit: for background and foreground, I think we can also put `.overflow` and `.outlined` on the same line, like `ringDefaultColor`.

Reply:
Yes! There are other instances of duplicated logic even on the ringDefaultColor (`aliasTokens.colors[.brandStroke1]`)

All of these repeated values can be consolidated in the same case of the switch statement:


```
    open var ringDefaultColor: DynamicColor {
        switch style {
        case .default, .group, .accent,  .outlinedPrimary:
            return aliasTokens.colors[.brandStroke1]
        case .outlined, .overflow:
            return aliasTokens.colors[.stroke1]
        }
    }
```